### ISSUE TITLE
Live USB demo 

### ISSUE THREAD

**Issue by zhovner** [zhovner/magic-setup-assistant#18]
I've made a USB image for testing our system.
It works in PC and Mac.
Check this out https://github.com/zhovner/magic-setup-assistant/wiki/Live-USB-image

**Comment by zhovner** [comment]
Let's test it @TheBeastOfCaerbannog @strangeworks

**Comment by diastremskii** [comment]
Oh wow, nice. 

I encountered this issues:

- It looks like USB boots only in UEFI mode, because BIOS boot partition is missing.  Creating BIOS boot partition and reinstalling GRUB for BIOS fixed it.
- Something is wrong with cyrillic fonts rendering and Chinese (or whatever that is 😀) is rendered as squares.
- Without VM it stuck at `fb: switching to nouveaufb from EFI VGA`. It was able to boot with nouveau.modeset=0, however, Xorg crashed. I ran it on desktop PC, so it shouldn't be a problem if it doesn't stuck on notebooks with Nvidia GPUs.

UPD: Oh, it doesn't stuck in BIOS mode. Xorg still crashed, however.

**Comment by zhovner** [comment]
> It looks like USB boots only in UEFI mode, because BIOS boot partition is missing.

Yes,  because i'm to lazy to manage MBR. Modern UEFI boot requires only regular fat32 partition in GPT scheme, so you can manage files on this partition with GUI like a regular USB drive, just drag&drop files into it. 

> Something is wrong with cyrillic fonts rendering and Chinese

Yeah. Don't know how to fix this yet.

> however, Xorg crashed. I ran it on desktop PC

What hardware do you have? I test it on macbook pro 15 2015 and Lenovo x220 and it works good.

**Comment by diastremskii** [comment]
> What hardware do you have? 

MSI 970A-G43 / AMD 1100T / GTX 970. 
I guess it should be tested on some notebook with Nvidia Optimus or something like that.

**Comment by diastremskii** [comment]
Oh, it seems I messed up with fonts. Cyrillic characters look different because current fonts doesn't have glyphs for them. Will update fonts ASAP.

**Comment by diastremskii** [comment]
Installing [fonts-noto-cjk](https://launchpad.net/ubuntu/+source/fonts-noto-cjk) should fix all other characters. That does make image a bit larger, however.
